evap/evaluation/views.py

```diff
@@ -94,3 +102,27 @@ def faq(request):
 
 def legal_notice(request):
     return render(request, "legal_notice.html", dict())
+
+
+@require_POST
+@login_required
+def feedback_send(request):
+    sender_email = request.POST.get("sender_email")
```

Comment:
you could add a disabled field showing the email address. then everybody knows, that the adress will be included.

OWA does not allow that, the emails have to be sent from our own account (evaluierung). the email adress should be included in the message.